README.md

```diff
@@ -29,26 +29,24 @@ These contract function calls should ultimately be encoded into a governance pro
 ## Usage
 
 This package vends a CLI for executing a deployment script that results in a full deployment of Uniswap Protocol v3.
-Get the arguments for running the latest version of the script via `npx @uniswap/deploy-v3 --help`.
+Get the arguments for running the latest version of the script via `yarn start --help`.
 
-As of `v1.0.3` the arguments are:
+As of `v1.0.4` the arguments are:
 
 ```text
-> npx @uniswap/deploy-v3 --help
-Usage: npx @uniswap/deploy-v3 [options]
-
-Options:
-  -pk, --private-key <string>               Private key used to deploy all contracts
-  -j, --json-rpc <url>                      JSON RPC URL where the program should be deployed
-  -w9, --weth9-address <address>            Address of the WETH9 contract on this chain
-  -ncl, --native-currency-label <string>    Native currency label, e.g. ETH
-  -o, --owner-address <address>             Contract address that will own the deployed artifacts after the script runs
-  -s, --state <path>                        Path to the JSON file containing the migrations state (optional) (default: "./state.json")
-  -v2, --v2-core-factory-address <address>  The V2 core factory address used in the swap router (optional)
-  -g, --gas-price <number>                  The gas price to pay in GWEI for each transaction (optional)
-  -c, --confirmations <number>              How many confirmations to wait for after each transaction (optional) (default: "2")
-  -V, --version                             output the version number
-  -h, --help                                display help for command
+Usage: yarn start [--confirmations <STRING>] [--gas-price <STRING>] --json-rpc <STRING> --native-currency-label <STRING> --owner-address <STRING> --private-key <STRING> [--state <STRING>] [--v2-core-factory-address <STRING>] --weth9-address <STRING>
+
+OPTIONS:
+
```

Comment:
Redundant line?

Reply:
That's actually just the output of `yarn start --help`, but we can remove this line for consistency